packages/rebox/web/src/build-web/get-config.ts

```diff
@@ -65,6 +65,16 @@ export const getConfig = (entryPointPath: string, outputPath: string, htmlTempla
           },
         ],
       },
+      {
+        test: /\.png$/,
+        exclude: /node_modules/,
+        loader: 'file-loader?name=images/[name].[ext]',
+      },
+      {
+        test: /\.mp4$/,
+        exclude: /node_modules/,
+        loader: 'file-loader?name=videos/[name].[ext]',
```

Comment:
`file-loader` should be an explicit dependency of Rebox.

Also, would be nice to do the same for the rest of methods like `runWeb` and native ones.

Reply:
I agree with you it makes to have it for native as well, but honestly I have no idea what's going on there :/
can you help me out with adding the native ones?

Reply:
sure :)

Reply:
thanks! 🙏